src/core/metadata/qgslayermetadata.cpp

```diff
@@ -158,6 +158,11 @@ void QgsLayerMetadata::addKeywords( const QString &vocabulary, const QStringList
   mKeywords.insert( vocabulary, keywords );
 }
 
+bool QgsLayerMetadata::removeKeywords( const QString &vocabulary )
+{
+  return bool( mKeywords.remove( vocabulary ) == 1 );
```

Comment:
I prefer bool, and `remove` returns the number of lines removed. In our case, it will always return one.